app/src/pages/trace/TracePage.tsx

```diff
@@ -379,15 +389,19 @@ function LLMSpanInfo(props: { span: Span; spanAttributes: AttributeObject }) {
           </TabPane>
         </Tabs>
       </TabbedCard>
-      {output && output.value != null ? (
-        <Card
-          title="Output"
-          {...defaultCardProps}
-          backgroundColor="green-100"
-          borderColor="green-700"
-        >
-          <CodeBlock {...output} />
-        </Card>
+      {hasOutput || hasOutputMessages ? (
+        <TabbedCard {...defaultCardProps}>
+          <Tabs>
+            {hasOutput ? (
+              <TabPane name="Output" hidden={!hasOutput}>
+                <CodeBlock {...output} />
+              </TabPane>
+            ) : null}
+            <TabPane name="Messages" hidden={!hasOutputMessages}>
+              <LLMMessagesList messages={output_messages} />
+            </TabPane>
+          </Tabs>
```

Comment:
I think we should swap the order of these so messages shows up first if it exists

Reply:
ok. And do the same with input_messages, right?